src/generate.js

```diff
@@ -94,7 +94,7 @@ export const defaultSelectorHandlers = [
         }
         // Generate the styles normally, and then wrap them in the media query.
         const generated = generateSubtreeStyles(baseSelector);
-        return `${selector}{${generated}}`;
+        return [`${selector}{${generated}}`];
```

Comment:
Hmmm. I think the funky `@media all{}` with a warning solution sounds the best to me, but I'm not that attached to it. I don't know how many people are using extensions right now, but I'd prefer if we didn't break it already?

(and as a ping, the `psuedoSelectors` function above needs to be changed too, right?)